### DISCUSSION TITLE
关于设置表单中select 选项的 disabled 属性问题，建议增加一个API

### DISCUSSION THREAD

**Discussion by HuangLei0x2e5** [layui/layui#1904] [建议]
我在表单中要按一定条件来决定某个 select 是否可用，是通过
      layui.$("#id").attr("disabled", "disabled");
      layui.$("#id").removeAttr("disabled");
来切换，发现无效，后来折腾好久，又想到上次也碰到类似的问题，发现要这样才行：
      layui.$("#id").attr("disabled", "disabled");
      layui.form.render('select');
      layui.$("#id").removeAttr("disabled");
      layui.form.render('select');
      
 是不是在表格属性里增加一个API来简化操作：
 
 alyui.form.disabled("my-filter",true)

**Comment by Sight-wcg** [comment]
自行封装一下吧
```
function updateLayFormElementAttributes(el, attrObj){
  var elem = $(el);
  elem.attr(attrObj);
  layui.form.render(elem);
}
```
